testsuite/features/support/code_coverage.rb

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) 2016-2023 SUSE LLC.
+# Licensed under the terms of the MIT license.
+require 'redis'
+require 'nokogiri'
+require 'open-uri'
+
+# CodeCoverage handler to produce, parse and report Code Coverage from the Jave Server to our GitHub PRs
+class CodeCoverage
+  include(Nokogiri::XML)
+  ##
+  # Initialize a connection with a Redis database
+  def initialize(redis_host, redis_port, redis_username, redis_password)
+    @database = Redis.new(host: redis_host, port: redis_port, username: redis_username, password: redis_password)
+  end
+
+  ##
+  # Close the connection with the Redis database
+  def close
+    @database.close
+  end
+
+  ##
+  # Parse a Jacoco XML report, extracting information that will be included in a Set on a Redis database
```

Comment:
```suggestion
  # Parse a JaCoCo XML report, extracting information that will be included in a set on a Redis database
```

Reply:
Set is a concrete kind of Redis type.
https://redis.io/docs/data-types/sets/

But ok :)

Reply:
I am ware that you do not mean the verb `set` here but rather the data structure. Nevertheless it is written without capital letter 😉